photon-server/src/main/java/org/photonvision/common/hardware/metrics/MetricsBase.java

```diff
@@ -53,11 +58,15 @@ public static void setConfig(HardwareConfig config) {
         ramUsageCommand = config.ramUtilCommand;
```

Comment:
Here you'll also want to set the `diskUsageCommand `

Reply:
Good catch - Added into the HardwareConfig class as well